### DISCUSSION TITLE
Account got suspendes and can't send mail to support@github too

### DISCUSSION THREAD

**Discussion by Arsallll** [community/community#116559] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi.
I got my account suspended suddenly without doing anything wrong.
please help me get back my account
Acooutn -- https://github.com/arsalanash

**Comment by VictoriqueMoe** [comment]
I know multiple other people this happanes to as well, R.E: https://github.com/nakedmcse

**Reply by Arsallll** [reply]
do you know the solution to fix this

**Reply by gpereiradev** [reply]
I have the same problem

**Reply by arvindrag** [reply]
Same issue

**Reply by VictoriqueMoe** [reply]
Did not happen to me, but people i know, someone said that resetting a password might work, but email won't be recognised. try use `username` instead

**Reply by arvindrag** [reply]
'reset password' sends you back to Account Suspended page

**Reply by Arsallll** [reply]
@VictoriqueMoe there is no option for try use username
@arvindrag Yeah

**Comment by nakedmcse** [comment]
I had to reset my password using my email account and not the username.
After I did that I could get back in.

**Reply by Arsallll** [reply]
after i try to reset my password from the link that I got on mail 
It sends back account suspended page

**Reply by nakedmcse** [reply]
I never got the "account suspended" page, my account simply disappeared and then reappeared.

So this may be a different problem.

**Reply by Arsallll** [reply]
I got my account suspended and when I try to see my account pages it says error 404 account not found.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Arsallll ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
